docs/azure.md

```diff
@@ -0,0 +1,43 @@
+### Azure
+
+### Export your credentials as environment variables
+
+[Create Azure Service Principal](https://www.terraform.io/docs/providers/azurerm/authenticating_via_service_principal.html) then export the credentials.
+
+```bash
+$ export AZURE_CLIENT_ID=YOUR_AZURE_CLIENT_ID
+$ export AZURE_CLIENT_SECRET=YOUR_AZURE_CLIENT_SECRET
+$ export AZURE_SERVICE_PRINCIPAL=YOUR_AZURE_SERVICE_PRINCIPAL
+$ export AZURE_SUBSCRIPTION_ID=YOUR_AZURE_SUBSCRIPTION_ID
+$ export AZURE_TENANT_ID=YOUR_AZURE_TENANT_ID
+```
+
+## Create the infrastructure
+
+### Initialize the packer environment
+
+```bash
+$ make terraform-init provider=azure env=images-builder
```

Comment:
I'm not aware of details on the project etc, but it seems a little off that the description says that you are going to initialize the packer environment but the make target is "make terraform-init". It seems like some leaking of the internal tool used to do stuff. If i'm initializing a packer env I would expect "make packer-env" (terraform is the detail). But it is just some food for thought, perhaps there is good reasoning to keep it this way.

Reply:
Thanks for the advice! I'll explain trough a better way what we doing here.